Projects/UOContent/Gumps/Props/PropsGump.cs

```diff
@@ -551,12 +478,12 @@ private static string[] GetCustomEnumNames(Type type)
             return ce.Names;
         }
 
-        public static bool HasAttribute(Type type, Type check, bool inherit) =>
+        private static bool HasAttribute(Type type, Type check, bool inherit) =>
```

Comment:
Lots of these static `Type` related functions can probably go into `Server.Types` and stay public so they can be reused. This way we can create more `Props` stuff without having to recreate them.

Reply:
Moved this to a newly created `TypesAttributes.cs` to separate `Type` constants and helpers that access attributes via reflection in acfc5ed.